presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -302,6 +302,18 @@ public void createPartitionedTable(Session session, HiveStorageFormat storageFor
 
         assertUpdate(session, "INSERT INTO test_partitioned_table " + select, 1);
         assertQuery(session, "SELECT * from test_partitioned_table", select);
+        assertQuery(session,
+                "SELECT * from test_partitioned_table WHERE" +
+                        " 'foo' =  _partition_string" +
```

Comment:
nit: two spaces after `=` on L307, L309, L312, L313, and no space after `=` on L315.

also seems like in this class we are not consistent with the spacing around `CAST` calls.